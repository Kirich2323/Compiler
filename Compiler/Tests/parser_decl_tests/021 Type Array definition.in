type
    myArrayType = Array [1..2] of integer;