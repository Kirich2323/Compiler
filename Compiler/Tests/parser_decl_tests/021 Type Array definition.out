main
----------
char                     type 
float                    type 
integer                  type 
myArrayType        type alias array [1..2] of integer